### DISCUSSION TITLE
Multiple Previous and Next buttons in Carousel: Headings and Labels: Understanding SC 2.4.6

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#3149] [Q&A] [answered]
Hi Everyone,

I have encountered a scenario on a web page that requires your attention regarding its compliance with WCAG 2.4.6. The page contains multiple carousels, each having its own set of previous and next buttons. However, there is a concern that screen reader users may face difficulty in identifying which previous and next buttons belong to which carousel.

While it is evident that the functionality of the previous and next buttons is the same.
And the context is the same here.

Is it still essential to provide descriptive programmatic labels for each set of previous and next buttons? (Example Previous slide of carousel A, Next Slide of carousel A, Previous slide of carousel B, Next slide of carousel B, and so on)

Please share your thoughts.

**Comment by AccessibilityEnthusiastic** [comment]
@patrickhlauke @JAWS-test
Can you please provide insight on the above issue?

**Reply by JAWS-test** [reply, marked as answer]
I'm not sure if it would be a violation of 2.4.6 to have the buttons labeled only Previos and Next. 
In any case, it makes sense to label the buttons in such a way that their purpose can be clearly identified. This can also be done implicitly by using labeled groups or regions (role=group, role=region).
